src/components/ErrorBoundary/ErrorPopup.tsx

```diff
@@ -33,11 +42,32 @@ const ErrorPopup = ({
 }: IErrorPopupProps) => {
   const [expandMessageBar, setExpandMessageBar] = React.useState(false);
   const hasMoreErrors = errorCount > 0;
+
+  function getErrorStack(): StackProperties | undefined {
+    if (!currentError.error.stack) {
+      return undefined;
+    }
+    const parsedErrorStack: StackProperties = JSON.parse(
+      currentError.error.stack
+    );
+    return parsedErrorStack;
+  }
+
+  const errorStack = getErrorStack();
+
+  const isAppRereshFailedCode = errorStack?.['code'] === 'APP_REFRESH_FAILED';
+
+  const ad = errorStack?.['applicationDeployment'];
```

Comment:
code kan vel være en del av StackProperties?

Er det noen grunn for å hente applicationDeployment med ` errorStack?.['applicationDeployment']` fremfor ` errorStack?.applicationDeployment`?

Reply:
Jepp, skal ta å legge den til i interfacet, sjekket også at den skal bare bli string.

Syntes det ble litt mer consistet enn å ha errorStack?.['code'] og errorStack?.applicationDeployment.
Men nå som jeg får type info til begge, så syntes jeg absolutt det er bedre med errorStack?.applicationDeployment og errorStack?.code :)